cmd/quickstart/quickstart.go

```diff
@@ -0,0 +1,32 @@
+package quickstart
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	var quickstartcmd = cobra.Command{
+		Use:   "quickstart [sdk]",
+		Short: "Create a minimal getting started app for the specified SDK",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			sdk := args[0]
```

Comment:
lets just add a check in case they don't add any arg so we don't panic.

Reply:
Yes, you're correct. The Args: cobra.ExactArgs(1) configuration in Cobra ensures that the command expects exactly one argument. If the user doesn't provide exactly one argument when executing the command, Cobra will display an error message and the command won't be executed.

Reply:
I like it can u create an issue to track doing this for all the other commands for consistency?

Reply:
if you wanna just do it feel free too.